static/app/views/dashboards/widgetBuilder/buildSteps/thresholdsStep/thresholdsHoverWrapper.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import type React from 'react';
```

Comment:
Would be great to figure out how to ban this import and make everyone use he global `React.*` type

Reply:
It's quite easy. I can solve it in a follow-up PR.